react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { count: 0 };
+    this.numberClick = this.numberClick.bind(this);
+  }
+
+  numberClick() {
+    this.setState({ count: this.state.count + 1 });
+  }
+
+  render() {
+    let className = 'white';
+    const count = this.state.count;
+    if (count >= 3 && count < 6) {
```

Comment:
Make sure to move this logic to a separate method as in the previous challenge to clean up the render method.